.github/workflows/ci.yaml

```diff
@@ -11,86 +11,104 @@ env:
 
 jobs:
   # Run unit tests.
-  test:
-    name: Test Suite
+  unit-test:
+    name: Unit Tests
```

Comment:
Misnomer: also includes integration tests, regression tests, smoke tests, benchmarks, etc.
This comment is for all instances of this rename

Reply:
Alright. What should I call this to distinguish it from doc tests?

Reply:
But it also runs doc tests! The second one is more of a `Check Docs` or `Validate Docs`

Reply:
Okay. The manual page was confusing:
```
       --all-targets
           Test all targets. This is equivalent to specifying --lib --bins --tests --benches --examples.

       --doc
           Test only the library’s documentation. This cannot be mixed with other target options.
```

Reply:
Wait, hold up. You're right, it does not run doc tests

Reply:
`Tests` vs `Doc Tests` is good for me then.

Reply:
https://github.com/rust-lang/cargo/issues/6669 :smiling_face_with_tear: